blocks/bio-detail/bio-detail.css

```diff
@@ -1,12 +1,24 @@
-.bio-detail .resources ul {
-  list-style: none;
-  padding: var(--none);
+.bio-detail .resources p a {
+  margin: 0;
+  border-bottom: 2px solid transparent;
+  padding: 4px 0;
+  font-weight: 400;
 }
 
-.bio-detail .resources ul li a {
-  color: var(--primary-blue-70);
-  text-decoration: none;
-  font-size: var(--body-medium);
+.bio-detail .media-inquiries p a{
+  padding: 0;
+  margin: 0;
+  border-bottom: 2px solid transparent;
+  color: #000D1A;
+}
+
+.bio-detail .media-inquiries p a:hover{
+  color: #000D1A;
```

Comment:
both normal and hover colors are same ? if so we dont need this

Reply:
on hover the color was is getting overridden from the styles file. so adding this @rrrragas